account_move_line_purchase_info/hooks.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def post_init_hook(cr, registry):
+
+    """ INIT purchase_line_id in acount move line """
+    # FOR stock moves
+    cr.execute(
+        """
+        UPDATE account_move_line aml SET purchase_line_id = sm.purchase_line_id
+        FROM account_move_line aml2
+        INNER JOIN account_move am
+        ON am.id = aml2.move_id
+        INNER JOIN stock_move sm ON
+        am.stock_move_id = sm.id
+        WHERE aml.id = aml2.id
+        AND sm.purchase_line_id IS NOT NULL;
```

Comment:
```suggestion
        AND sm.purchase_line_id IS NOT NULL AND aml.purchase_line_id IS NULL;
```
same in the migration query

Reply:
@JordiBForgeFlow you have here a suggestion

Reply:
Actually, I think the intention is to overwrite the existing values in the column purchase_line_id. Theoretically, the values in purchase_line_id are already correct if this module is already installed in the database.